zone/spell_effects.cpp

```diff
@@ -6839,11 +6828,9 @@ void Mob::CheckNumHitsRemaining(NumHit type, int32 buff_slot, uint16 spell_id)
 			if (IsValidSpell(buffs[d].spellid) && buffs[d].hit_number > 0 &&
 			    spells[buffs[d].spellid].hit_number_type == static_cast<int>(type)) {
 
-#ifdef BOTS
 				buff_name = spells[buffs[d].spellid].name;
```

Comment:
This seems to change how this logic fundamentally works when bots are enabled, at the very least rule gate this still or lets ask why this behaves differently when bots are enabled because it affects any mob

Reply:
Same as above.